expression/builtin_string_vec_test.go

```diff
@@ -28,6 +28,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.Lower: {
 		{types.ETString, []types.EvalType{types.ETString}, nil},
 	},
+	ast.IsNull: {
+		{types.ETInt, []types.EvalType{types.ETString}, []dataGenerator{&randLenStrGener{10, 20}}},
```

Comment:
`randLenStrGener` will not generate `null` data, so please add another case to test it. (Using the `defaultGener` maybe OK)